File: sistema_facturacion_api/sistema_facturacion_api.Useful/Filtros/FiltroUsuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sistema_facturacion_api.Useful.Filtros
{
    public class FiltroUsuario : ParametrosDeBusqueda
    {
        public string? Identificacion { get; set; }
        public string? Pasaporte { get; set; }
    }
}

File: sistema-facturacion-front/src/app/models/dtos/filtro-usuario.ts
import { ParametrosDeBusqueda } from "./parametros-de-busqueda";


export class FiltroUsuario extends ParametrosDeBusqueda{
  Identificacion: string;
  Pasaporte: string;

  constructor() {
    super();
    this.Identificacion = '';
    this.Pasaporte = '';
  }
}
